__mocks__/fileMock.js

```diff
@@ -0,0 +1 @@
+module.exports = 'test-file-stub';
```

Comment:
Dette er bare for at Jest ikke skal feile på import av filer den ikke skjønner (som svg osv). Se `moduleNameMapper` under `libs/ui/jest.config.js`. Samme med `identify-obj-proxy` som gjør at Jest tåler import av CSS-moduler.